CMakeLists.txt

```diff
@@ -652,7 +654,7 @@ int main() {return 0;}" HAVE_SZIP_WRITE)
 ###
 # Check to see if quantize capability is present in netcdf-c.
 ###
-SET(CMAKE_REQUIRED_INCLUDES ${NETCDF_INCLUDE_DIR})
+SET(CMAKE_REQUIRED_INCLUDES ${NETCDF_C_INCLUDE_DIR})
```

Comment:
Are we currently testing on windows in the CI?

Reply:
In the CI?  Not sure what you mean.  All changes were made on a Windows system using CMake.